pkgs/development/python-modules/knowit/matroska_tests.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/tests/__init__.py b/tests/__init__.py
+index 5c91d3b..7faad8d 100644
+--- a/tests/__init__.py
++++ b/tests/__init__.py
+@@ -5,10 +5,8 @@ import re
+ import sys
+ from collections.abc import Mapping
+ from datetime import timedelta
+-from io import BytesIO
+ from zipfile import ZipFile
+ 
+-import requests
+ import yaml
+ from yaml.constructor import Constructor
+ 
+@@ -103,8 +101,7 @@ class MediaFiles(object):
+ 
+         # download matroska test suite
+         if not os.path.exists(data_path) or len(os.listdir(data_path)) != 8:
```

Comment:
I think that according to this conditional, we should be able to avoid maintaining this patch, and use Nix's `fetchzip` and copy the files to this directory. This would make the future maintenance much easier. Could you please give it a try?